docs/topics/dev-guide-mission-intro.adoc

```diff
@@ -15,6 +17,13 @@ You can use missions for a variety of purposes:
 * They can also be updated or extended for your own use case
 
 .What is a Booster?
-A booster is the implementation of a mission in a specific runtime. Boosters are preconfigured, functioning applications that demonstrate a fundamental aspect of modern application development run in an environment similar to production.
+A booster is the implementation of a mission in a specific runtime. Boosters are preconfigured, functioning applications that demonstrate a fundamental aspect of modern application development and run in an environment similar to production.
+
+Each mission is implemented in one or more runtimes. Both the specific implementation and the actual project that contains your code are called a booster.
+
+For example, the {name-mission-http-api} mission has support for boosters and runtimes such as:
```

Comment:
This sentence sounds weird :-) I'd say something like "For example, the {name-mission-http-api} mission is implemented for these runtimes:".

Reply:
@Ladicek I'll make that change.